### DISCUSSION TITLE
Old Plugins Causing 404 Errors

### DISCUSSION THREAD

**Discussion by josh99smith** [Budibase/budibase#12222] [Help] [answered]
Does anyone have an idea why my Budibase instance keeps looking for any old plugins that I have installed? This happen even after I remove them from budibase plugins settings page, the app still wants to inject them into my pages. This is is true for ANY plugin I installed. Even after removing it, budibase will continue to search for it.  Here is a picture of what the console is showing. These are all plugins that I am no longer using, however Budibase keeps looking for them on everypage of my app.

![image](https://github.com/Budibase/budibase/assets/4806827/4e3d6b82-3a46-499c-b0f6-d9e1a2b105bb)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @josh99smith,

There is an open issue related to [this](https://github.com/Budibase/budibase/issues/10161). These console errors are harmless and shouldn't do anything to affect your app. The only way currently to completely remove these is to export the app, delete the app and re-import the app at the moment. I've linked the related issue earlier, for additional information have a read through it.

I hope this helps.

**Reply by josh99smith** [reply]
Thank you so much for clearing that up! Just making sure there wasnt an easy fix out there.
